tests/test_memory_proxy_wrapper.py

```diff
@@ -0,0 +1,80 @@
+import pytest
+
+from emote.memory.memory import MemoryProxyWrapper
+
+
+class DummyMemoryProxy:
+    def __init__(self):
+        self.my_attribute = "hello!"
+
+    def say_hello(self):
+        return "hello world"
+
+
+class EmptyMemoryProxyWrapper(MemoryProxyWrapper):
+    def __init__(self, inner):
+        super().__init__(inner)
```

Comment:
This is an anti-pattern - just `pass` if the body is meant to be empty.

Reply:
Copy that!